docs/installation/docker.md

```diff
@@ -0,0 +1,22 @@
+# Installing Loki with Docker/Docker-compose
+
+## Prerequisites
+
+Basiclly, you must install docker.
+If you want to install loki with docker-compose, install it too.
+
+## Install with Docker
+
+```bash
+$ wget https://raw.githubusercontent.com/grafana/loki/master/cmd/loki/loki-local-config.yaml -o loki-config.yaml
+$ docker run -p 3100:3100 grafana/loki:latest -config.file=./loki-config.yaml
+$ wget https://raw.githubusercontent.com/grafana/loki/master/cmd/promtail/promtail-docker-config.yaml -o promtail-config.yaml
+$ docker run -v /var/log:/var/log grafana/promtail:latest -config.file=./promtail-config.yaml
```

Comment:
Few things here:

1. Even though the files are being downloaded, they're not being sent to the container. A volume mount is needed. something like `-v $(pwd):/mnt/config -config.file=/mnt/config/loki-config.yaml` 
2. Rather than using latest, I think we should refer to the latest release tag here. We'll keep it up to date as part of our release process.